vue-app/src/pages/GridPage.vue

```diff
@@ -0,0 +1,345 @@
+<template>
+  <div class="page-wrapper">
+    <div class="page-content">
+      <div :style="{ width: '100%' }">
+        <div class="windows-container">
+          <grid-button-window title="Aspects">
```

Comment:
those elements are so nested, isnt it possible to reduce this div tree?

Reply:
same as in react PR